ch.openboard.OpenBoard.appdata.xml

```diff
@@ -19,6 +19,7 @@
     </screenshot>
   </screenshots>
   <releases>
+    <release version="1.5.1" date="2018-12-13" type="stable"></release>
```

Comment:
It's just an XML file, if you don't have any description to add for the release changelog just use
`    <release version="1.5.1" date="2018-12-13" type="stable" />`

Reply:
Fixed